syntax/std.go

```diff
@@ -51,8 +51,12 @@ func StdScope() rel.Scope {
 					case rel.Dict:
 						attrs := make([]rel.Attr, 0, t.Count())
 						for e := t.DictEnumerator(); e.MoveNext(); {
-							key, value := e.Current()
-							attrs = append(attrs, rel.NewAttr(key.(rel.String).String(), value))
+							keyVal, value := e.Current()
+							key := ""
+							if keyVal.IsTrue() {
```

Comment:
need to check if `keyVal` is not `0` or `()`. It won't catch `//tuple({0: 0, (): ()})`